### ISSUE TITLE
sessionDelay does not exist

### ISSUE THREAD

**Issue by bangsyir** [cyrildewit/eloquent-viewable#132]
hello, i try use Recording views with

```php
views($post)
    ->sessionDelay(now()->addHours(2))
    ->record();
```

but i get an error in the browser, "class sessionDelay does not exist", 
can you help me please

**Comment by cyrildewit** [comment]
Ohh, my mistake. It's `->delayInSession()`.

So:

```php
views($post)
    ->delayInSession(now()->addHours(2))
    ->record();
```

**Comment by bangsyir** [comment]
oh thanks for help me. now i have new problem. The code keeps recording every time i'm refresh my browser.
views($post)
    ->delayInSession(now()->addHours(2))
    ->record();

now()->addHours(2) this code not function...

**Comment by cyrildewit** [comment]
Hey @bangsyir,

This should be fixed in `v3.0.2`! Let me know if it's still not working properly!
